ase/users/models.py

```diff
@@ -7,6 +7,15 @@
 # Create your models here.
 
 class user(AbstractUser):
+    """
+    -- Summary --
+    User model has the details of the user.
+    #Field Descriptions
+    > username is a character field with maximum possible length = 25 which is basically the username of the user and is unique for every user.
+    > email is the email id of the user
```

Comment:
Refer to this comment for changes
https://github.com/Sayam753/ASEProject/pull/1/files#r456696110